apps/modules/home/screen_home.ios.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ */
+
+import React, {
+  AppRegistry,
+  Component,
+  StyleSheet,
+  Text,
+  View,
+  TouchableHighlight,
+  AlertIOS
+} from 'react-native';
+
+let navigator;
+
+class HomeScreen extends Component {
+  
+  constructor(props) {
+    super(props);
+    navigator = props.navigator;
+  }
+  
+  logout() {
+        navigator.replace({
```

Comment:
Remove `navigator.replace()` and replace it with this code block:
`
    this.props.navigator.replace({
      id: 'LoginScreen'
    });
`